app/views/admin/editions/_history_state.html.erb

```diff
@@ -1,8 +1,24 @@
 <% if edition.superseded? %>
-  <div class="alert alert-danger">
-    <p>This edition has been superseded.</p>
-    <p><%= link_to "Go to most recent edition", admin_edition_path(edition.latest_edition), class: "btn btn-primary" %></p>
-  </div>
+  <% if edition.latest_edition.id == edition.id %>
+    <div class="alert alert-info">
+      <p>
+        This edition has been superseded, you can still create a
+        new draft for this document.
+      </p>
+
+      <%= button_to(
+        "Create new edition to edit",
+        revise_admin_edition_path(edition),
+        title: "Create new edition to edit",
+        class: "btn btn-primary add-top-margin"
```

Comment:
I'm not entirely sure, but this action is a little different. The others are just links, styled as buttons, whereas this is a form styled as a button. and it's a form because the POST request to the server is actually creating the new draft.

Reply:
It looks good in the [screenshot you provided](https://github.com/alphagov/whitehall/pull/5630#discussion_r432378505), so happy to approve 👍